### ISSUE TITLE
Hash login password before comparing it to the stored password

### ISSUE THREAD

**Issue by zsoerenm** [zombor/Auto-Modeler#52]
The password is compared to the stored password, before the it's hashed:
https://github.com/zombor/Auto-Modeler/blob/master/classes/auth/automodeler/orm.php#L89

Shouldn't it be like this?:
https://github.com/kohana/auth/blob/3.2/master/classes/kohana/auth/file.php#L40

**Comment by zombor** [comment]
Yes: https://github.com/zombor/Auto-Modeler/blob/develop/classes/auth/automodeler/orm.php#L89

**Comment by zsoerenm** [comment]
Alright that's because of api change from 3.1 -> 3.2

**Comment by zsoerenm** [comment]
Auths hash function only accepts 1 parameter, it gets the salt from the config.

https://github.com/zombor/Auto-Modeler/blob/develop/classes/auth/automodeler/orm.php#L88
